src/Microsoft.Azure.WebJobs.Host/ExponentialBackoffRetryAttribute.cs

```diff
@@ -13,7 +13,9 @@ namespace Microsoft.Azure.WebJobs
     /// </summary>
     public class ExponentialBackoffRetryAttribute : RetryAttribute
     {
-        private IDelayStrategy _delayStrategy;
+        private TimeSpan _parsedMinimumInterval;
+        private TimeSpan _parsedMaximumInterval;
+        private static string _delayStrategyKeyName = "MS_DelayStratagy";
```

Comment:
nit: Move this to constants?

Reply:
discussed with Mathew, you are ok to leave it here